pkg/config/apm.go

```diff
@@ -164,6 +164,15 @@ func setupAPM(config Config) {
 		}
 		return out
 	})
+
+	config.BindEnv("apm_config.peer_tags", "DD_APM_PEER_TAGS")
```

Comment:
Consider adding this to `pkg/flare/envvars.go` if you want it included in an Agent Flare (which I think you probably do?)